crates/core/tedge_mapper/src/collectd_mapper/collectd.rs

```diff
@@ -273,25 +334,23 @@ mod tests {
     }
 
     #[test]
-    fn invalid_collectd_payload_more_separators() {
-        let payload = "123456789:98.6:abc";
```

Comment:
It might be good to add a unit test with an invalid list of values as in `123456789:98.6:abc`.

Reply:
added a test